### DISCUSSION TITLE
Add a dropdown to quick switch user

### DISCUSSION THREAD

**Discussion by sabersalv** [jesec/flood#243] [Ideas]
Right now you have to logout and login to switch to another user. Can we support multiple users? Can switch to a different user with a dropdown. Thanks.

**Comment by jesec** [comment]
Well. Even Microsoft and Google do it poorly. Github does not have the feature at all. 

In a word, it is difficult to implement quick user switching. You should let your browser remember multiple accounts, so you can use autocomplete to quickly login to a certain user. 

I am open to the multi-client inside one user idea. Though, that would be difficult to implement, as well.

**Reply by musicderp** [reply]
I would also very much appreciate one of these 2 features, multiple clients in 1 user sounds fantastic
